workflow/Snakefile

```diff
@@ -282,3 +279,32 @@ rule run_MotEvo_analysis:
         {input.DIR_pwm}) \
         1> {log.LOG_local_stdout} 2> {log.LOG_local_stderr}
         """
+
+rule combine_MotEvo_results:
+    """
+    Combine all motevo results in one tsv file
+    """
+    input: 
+        DIR_MotEvo_results_pwm = expand(
+            os.path.join(
+                "{outdir}",
+                "motevo",
+                "{pwm_dir}"
+            ),
+            outdir = config["outdir"],
+            pwm_dir = gather_motifs_names(config["pwm_directory"])
+        )
+    output:
+        combined_MotEvo_results = expand(os.path.join(
+            "{outdir}",
+            "combined_MotEvo_results.tsv"
+        ),
+        outdir = config["outdir"]
+        ),
+    params:
+        Input_filename = "posterior_sites"
+    shell: 
+        "python ../../../scripts/parse.py \
```

Comment:
This is a comment for @AngryMaciek :
Double check how to encode relative paths, also - check the execution bash scripts for `cwd` there

Reply:
This is a more complex issue - I will have to create a separate feature branch for this fix.